client/verta/docs/tutorials/deployment/endpoint_update.rst

```diff
@@ -0,0 +1,58 @@
+Model Deployment
+================
+
+After an Endpoint has been `created <endpoint_creation.html>`__, it needs to be updated with a
+machine learning model that can serve predictions.
+
+This tutorial demonstrates how to take a model you've logged with Verta and deploy it to an
+Endpoint.
+
+Using the Client
+----------------
+
+Whether the Endpoint is already live and serving predictions, or was newly created moments ago, the
+process for updating it with a new model is the same:
+
+.. code-block:: python
+
+    from verta.deployment.update import DirectUpdateStrategy
+
+    endpoint.update(model_version, DirectUpdateStrategy())
+
+The first argument to :meth:`Endpoint.update() <verta._deployment.endpoint.Endpoint.update>` is your
+:class:`~verta._registry.modelversion.RegisteredModelVersion` that already has a trained model and
+its Python environment logged.
+
+The second argument is a strategy to use for the update. Here, you would be using a simple
+:class:`~verta.deployment.update._strategies.DirectUpdateStrategy` that will fully transition the
+endpoint to use your new model. There are also more advanced ways of updating an endpoint, which
+you can read more about at the bottom of this tutorial.
+
+You can also update an Endpoint with an :class:`~verta._tracking.ExperimentRun`:
+
+.. code-block:: python
+
+    endpoint.update(run, DirectUpdateStrategy())
+
+Using the CLI
+-------------
+
+The same action can be done through the CLI. Again, using a model version and a direct update:
+
+.. code-block:: sh
+
+    verta deployment update endpoint /some-path --run-id <id> --strategy direct
```

Comment:
Add quotes around `<` and `>` when giving bash examples. Otherwise people can copy, this becomes a redirection and people will be confused.